template.tpl

```diff
@@ -70,14 +67,34 @@ const createQueue = require('createQueue');
 const blackcrow = copyFromWindow('blackcrow') || createQueue('blackcrow');
 callInWindow('blackcrow.push', {
     trigger: 'set',
-    data: {
+    data: {  
         member_id: data.member_id || ''
     }
 });
 
-let site_name = data.site_name;
 
-injectScript('https://init.blackcrow.ai/js/core/' + encodeUriComponent(site_name) + '.js?source=gtm&version=template');
```

Comment:
I think we want to keep source=gtm and version=template

Reply:
Sandbox doesn't like query string here anymore, not sure why...

![image](https://user-images.githubusercontent.com/96016324/159265294-25a0e273-ae75-4243-9a19-e60b5426adf0.png)